FogCreek/Glitch-Community
# Glow cheese: componentize FeaturedProjectOptionsPop

## Links
* Remix link: https://glow-cheese.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328624/

## GIF/Screenshots:
![Screen Shot 2019-05-23 at 4 08 32 PM](https://user-images.githubusercontent.com/938722/58282989-3710bc00-7d75-11e9-8412-28e873c6566f.png)

## Changes:
* componentize FeaturedProjectOptionsPop
* use AnimationContainer to animate un-featuring project

## How To Test:
* log into https://glow-cheese.glitch.me
* create a collection
* add a project to the collection
* click on "feature project" in the project's menu
* clicking on the featured project's menu should show its menu options
* clicking on "add note" should make a new note bubble appear
* clicking on "unfeature project" should make the featured project embed animate out and return to the projects list
